<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mediaDetailWidget</class>
 <widget class="QWidget" name="mediaDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>491</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pixmapLabel">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>pixmap</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yearLabel">
        <property name="text">
         <string>year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <widget class="QWidget" name="audiobook">
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>231</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="readerLabel">
             <property name="text">
              <string>reader</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="summaryButton">
             <property name="text">
              <string>read summary</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="podcast">
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>231</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="episodeLabel">
             <property name="text">
              <string>episode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="seasonLabel">
             <property name="text">
              <string>season</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="music">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>231</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="albumLabel">
             <property name="text">
              <string>album</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="lyricsButton">
             <property name="text">
              <string>read lyrics</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
